tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -40,6 +40,42 @@
 #include <regex>
 
 namespace {
+using AttributeKey_t = ROOT::Experimental::Detail::RDaosContainer::AttributeKey_t;
+using DistributionKey_t = ROOT::Experimental::Detail::RDaosContainer::DistributionKey_t;
+
+/// \brief RNTuple page-DAOS mappings
+enum EDaosMapping { kOClusterDColumn, kOUnique };
+
+struct RDaosKey {
+   daos_obj_id_t oid;
+   DistributionKey_t dkey;
+   AttributeKey_t akey;
+};
+
+/// \brief Some random distribution/attribute keys.
+static constexpr DistributionKey_t kDistributionKeyDefault = 0x5a3c69f0cafe4a11;
+static constexpr AttributeKey_t kAttributeKeyAnchor = 0x4243544b5344422a;
+static constexpr AttributeKey_t kAttributeKeyHeader = 0x4243544b5344422b;
+static constexpr AttributeKey_t kAttributeKeyFooter = 0x4243544b5344422c;
+static constexpr AttributeKey_t kAttributeKeyDefault = 0x4243544b5344422d;
+
+static constexpr daos_obj_id_t kOidMetadata{std::uint64_t(-3), 0};
+static constexpr daos_obj_id_t kOidPageList{std::uint64_t(-4), 0};
```

Comment:
Given that as of this PR the anchor, header, and footer are stored as different attribute keys, same OID (which breaks backwards compatibility with the previous code), maybe we could also "rename" these OIDs to `-1` and `-2`.